src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -184,35 +182,35 @@ export default class TacticSelectorContent extends Vue {
         break
     }
   }
-  tacticMenuColour (tactic: Tactic, item: TacticMenuItem) {
+  tacticMenuColour (tactic: Tactic, item: MenuItem) {
     if (item.action === TacticMenuOptions.PIN && tactic.pinned) {
       return 'primary'
     } else if (item.action === TacticMenuOptions.DELETE) {
       return 'error'
     }
     return ''
   }
-  tacticMenuIcon (tactic: Tactic, item: TacticMenuItem) {
+  tacticMenuIcon (tactic: Tactic, item: MenuItem) {
     return item.icon
   }
-  tacticMenuPinText (tactic: Tactic, item: TacticMenuItem) {
-    if (item.action === TacticMenuOptions.PIN && tactic.pinned && item.titleTwo) {
-      return item.titleTwo
+  tacticMenuPinText (tactic: Tactic, item: MenuItem) {
+    if (item.action === TacticMenuOptions.PIN && tactic.isPinned && item.titleTwo) {
```

Comment:
Does ` return bool ? true : false` make here more sense?

Reply:
No because we want to return text not a boolean

Reply:
I have done this if it was what you meant
```return item.action === TacticMenuOptions.PIN && tactic.isPinned && item.titleTwo ? this.$t(item.titleTwo) : this.$t(item.title)```